src/components/PopularMovie.tsx

```diff
@@ -1,7 +1,8 @@
 import { Link } from 'react-router-dom';
+import { API_IMG_URL } from 'src/constants/api';
 
 import useDeviceDetect from 'src/hooks/useDeviceDetect';
```

Comment:
please update this place too

Reply:
changed to relative imports. I basically use a plugin in VS Code that allows doing auto import, hence I didn't pay attention at once